### ISSUE TITLE
Add "supervised audit" enum to EventType

### ISSUE THREAD

**Issue by nmerket** [hpxmlwg/hpxml#26]
This is to support Home Energy Score. They have an assessment type that they need to indicate whether the audit was supervised by a "mentor". Their current mapping is [here](http://hescore-hpxml.readthedocs.org/en/latest/translation/building_address.html#assessment-type).

@gmdickey

**Comment by kevinwoley** [comment]
@gmdickey @nmerket - I would like to find a solution here that doesn't introduce new event types. This feels like it's a subproperty of an 'audit' that describes the type of audit. I'm concerned that the number of different event types would continue to expand, which makes it more difficult for the senders and receivers of HPXML to know what exactly they should be looking for when parsing a file. Most software doesn't care who the 'audit' was done by - they shouldn't have to think about this in their implementation.

**Comment by ejacobsohn** [comment]
If i understand what is being proposed, and I'm not speaking for HEScore, I
can see the identification of an audit as  "supervised" or somehow
including mentorship as being useful from a QA perspective. It might enable
one to see how effective their audit and mentorship programs work.

Kevin W.  when does software really care about anything? It tends to show
benign contempt for me on a regular basis. ;)

**Comment by kevinwoley** [comment]
@ejacobsohn - it's not just you that software shows contempt for. Just be glad you're not responsible for wrangling it. :)

I don't have a disagreement with the need/desire to indicate the information about the audit being supervised. That makes sense to me. My recommendation is to find a different way to communicate the information in HPXML other than the `EventType` element. For example, sending a file with two `Contractor` nodes and having the ability to mark one as the 'Supervisor' and the other as the 'Supervised' - which may provide for more useful, actionable information in the file. 

My goal here is to avoid changing the use cases implementation (which, unfortunately, are tied to the specific `EventType` set on the nodes.

In short, I'm of the opinion that adding new `EventType` enumerations should be avoided due to the complexity it introduces in the use cases and the software that supports them.

**Comment by nmerket** [comment]
@kevinwoley I'm fine with finding another way to specify that an audit was supervised. Possibly another element or something. I just need a way to get that bit of information out of the HPXML without using extensions like we are now. 

In HEScore, there end up being two audits: one for the mentor and one for the mentee. The mentor's audit shows up with the special "mentor" assessment type. The mentee just shows up as an initial audit.

**Comment by gmdickey** [comment]
I will weigh in, although the language is mostly foreign to me.  We do need to support the additional audit type.  How that is done, I will leave up to the software wranglers and their contemptuous product.  My only concern is that whatever change is implemented needs to be rolled out to the HEScore HPXML users in a controlled fashion so nothing breaks.

Glenn Dickey
Residential Buildings Technical Director
SRA International, Inc.
6003 Executive Boulevard, #400
Rockville, MD 20852
240-514-2859 (desk)

**Comment by nmerket** [comment]
You know what. I'm leaning towards leaving this as an extension as it is [currently implemented in the HEScore translator](http://hescore-hpxml.readthedocs.org/en/latest/translation/building_address.html#mentor-assessment-type). This just seems specific to the needs of HEScore.
